tests/unit/IncorporationApplication.ts

```diff
@@ -1,5 +1,5 @@
 import Vue from 'vue'
-import { getVuexStore } from '@/store'
+import { getVuexStore } from '@/stores'
```

Comment:
This doesn't look right on a couple of levels.

Please change this import, and rename this file to `IncorporationApplication.spec.ts`, and that's it. It can be fixed later when it's actually implemented.